src/libsyntax_pos/symbol.rs

```diff
@@ -942,6 +987,15 @@ pub mod kw {
 pub mod sym {
     use super::Symbol;
     symbols!();
+
+    // Get the symbol for an integer. Integers 0-9 have a static symbol and
+    // therefore are fast.
+    pub fn integer(n: usize) -> Symbol {
+        match digits_array.get(n) {
+            std::prelude::v1::Some(sym) => *sym,
+            std::prelude::v1::None => Symbol::intern(&n.to_string()),
```

Comment:
I don't think the `std::prelude::v1::` part is necessary, the function is not generated by a macro.

Reply:
Bare `Some` and `None` names caused problems because `sym::Some` and `sym::None` both exist. The compiler suggested `std::prelude::v1::`. I'd be happier with something shorter but I couldn't get anything else to work.

Reply:
Ah, I see.
Then this is a reasonable solution, `std::option::Option::Some` is another alternative.

Reply:
Or you can go esoteric:
```rust
match digits_array.get(n) {
    Some { 0: sym } => *sym,
    None {} => Symbol::intern(&n.to_string()),
}
```
:)

Reply:
`Option::Some` and `Option::None` worked, so I went with that.